api/comment_threads.rb

```diff
@@ -54,10 +54,15 @@
   filter_blocked_content params["body"]
   thread.update_attributes(params.slice(*%w[title body pinned closed commentable_id group_id thread_type]))
 
+  if params["user_id"] and value_to_boolean(params["read"])
+    user = User.only([:id, :username, :read_states]).find_by(external_id: params["user_id"])
+    user.mark_as_read(thread) if user
```

Comment:
@robrap Comment service is being used by multiple clients and initially there was no check for `user_id` or `read` params. This way it will simply ignore this part if these params are not passed to the endpoint; if we start throwing exception it might crash other clients. So i have tried to keep the default behaviour as is. Also as we are planning to rebuild comment service I believe we should not bring new changes for existing client for now.